bench/wtperf/runners/log.wtperf

```diff
@@ -1,10 +1,29 @@
+#
 # wtperf options file: Test performance with a log file enabled.
 # Set the log file small to catch log-swtich bottlenecks.
-conn_config="cache_size=1G,log=(enabled=true,file_max=200K),checkpoint=(log_size=500MB)"
+#
+# Perform updates instead of inserts to stress logging not eviction,
+# page splits or reconciliation.  Have it fit in cache.
+#
+# We expect this test can and will be run in other forms from the command
+# line to change log file size, pre-allocation, zero filling, logging off
+# and checkpoint off.
+#
+# Jenkins runs for perf testing:
+# - Config as-is
+# - Config + "-o "value_sz=1000" to test larger values and log records
+# - Config + "-C "log=(file_max=1M)" to test small log files and switching
+# - Config + "-C "log=(zero_fill=true,file_max=1M)" to test zero-filling
+# - Config + "-C "checkpoint=(wait=0)" to test no checkpoints
+# - Config + "-C "log=(prealloc=false,file_max=1M)" to test no pre-allocation
+# - Config + "-C "log=(zero_fill=true,file_max=1M)" to test zero-filling
```

Comment:
I'd feel more comfortable if these specified the full log configuration (i.e: include ``enabled=true``).